### ISSUE TITLE
The RawScript(byte[] invocationScript, String scriptHash) function should init verificationScript with byte[0].

### ISSUE THREAD

**Issue by chenquanyu** [neow3j/neow3j#78]
The serialize function will throw exception when verificationScript is null.

**Comment by wanglongfei88** [comment]
```
public RawScript(byte[] invocationScript, String scriptHash) {
        this.invocationScript = new RawInvocationScript(invocationScript);
        this.scriptHash = Numeric.hexStringToByteArray(scriptHash);
    }
```
When I use this method to create a `RawScript` object, `verificationScript` will be null. Eventually, `io.neow3j.crypto.transaction.RawScript.serialize` will throw `NullPointerException` exception. 

This is blocking construct `InvocationTransaction` to withdraw NEO or GAS from a contract.

**Comment by gsmachado** [comment]
This is published on the `2.1.0-SNAPSHOT`. It will be included on the `2.1.0` release.
